Automattic/wp-calypso
# Store: Add Stripe Connect Account to redux

In support of #17002 

Adds the actions, handlers, reducers and unit tests for Stripe Connect account creation

See also https://github.com/Automattic/woocommerce-services/pull/1137 for the corresponding WooCommerce Services PR, but that PR does not need to be merged for this PR (unit tests will pass - integration testing will be addressed in a subsequent PR that connects the UI to this)

To test:

Run `npm run test-client client/extensions/woocommerce` and ensure everything passes

cc @nabsul @v18
